test/Microsoft.ML.Tests/Transformers/ValueMappingTests.cs

```diff
@@ -506,6 +514,60 @@ public void ValueMappingWorkout()
 
             // Workout on value mapping
             var est = ML.Transforms.Conversion.ValueMap(keys, values, new[] { ("D", "A"), ("E", "B"), ("F", "C") });
+            var schema = est.GetOutputSchema(SchemaShape.Create(dataView.Schema));
+            foreach (var name in new[] { "D", "E", "F" })
+            {
+                Assert.True(schema.TryFindColumn(name, out var originalColumn));
+                Assert.Equal(SchemaShape.Column.VectorKind.Scalar, originalColumn.Kind);
+                Assert.Equal(NumberType.I4, originalColumn.ItemType);
+            }
```

Comment:
I dont think you need to retest this here. TestEstimatorCore will run call GetOutputSchema multiple times.